packages/jellyfish-api-core/__tests__/category/blockchain/getTxOutSetInfo.test.ts

```diff
@@ -0,0 +1,32 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+
+describe('TxOutSetInfo', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.generate(1)
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getTxOutSetInfo', async () => {
+    const txOutSetInfo = await client.blockchain.getTxOutSetInfo()
+
+    expect(txOutSetInfo).toStrictEqual({
+      height: 1,
+      bestblock: expect.any(String),
+      transactions: 10,
+      txouts: 15,
+      bogosize: 1117,
+      hash_serialized_2: expect.any(String),
+      disk_size: 0,
+      total_amount: new BigNumber(400000259.8)
+    })
```

Comment:
We can keep them static? They won't be flaky?

Reply:
I was wondering about this. It seems at least locally, the values besides `bestblock` and `hash_serialized_2` are always the same (depending on `nblocks`). Ran the test multiple times. 

However, perhaps I will change it to simply test for the type to just prevent the possibility of it being flaky.